src/astarte-client/types/token.ts

```diff
@@ -0,0 +1,23 @@
+/*
+   This file is part of Astarte.
+
+   Copyright 2020 Ispirata Srl
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+export type JWT = string;
+
+export type DecodedJWT = {
+  [key: string]: any;
```

Comment:
To describe and interact with what an Astarte token is, we'll require a dedicated model, this provides almost no validation and requires the usage of an utils file (see comment below).
I'd rather postpone this to a different PR and leave the parent variable as `:any`.